src/control/server/config_test.go

```diff
@@ -306,6 +306,19 @@ func TestConfigValidation(t *testing.T) {
 			},
 			msgBadConfig + relConfExamplesPath + ": " + msgConfigBadAccessPoints,
 		},
+		"single access point no port": {
+			func(c *Configuration) *Configuration {
+				return c.WithAccessPoints("1.2.3.4")
+			},
+			"",
+		},
+		"single access point invalid port": {
+			func(c *Configuration) *Configuration {
+				return c.WithAccessPoints("1.2.3.4").
+					WithControlPort(0)
+			},
+			msgBadConfig + relConfExamplesPath + ": " + msgConfigBadPort,
+		},
```

Comment:
One case you could add: With access point "1.2.3.4:0", you would hit the error message as well.

Reply:
sure thing